amass/sources/twitter_test.go

```diff
@@ -0,0 +1,38 @@
+package sources
+
+import (
+	"testing"
+	
+	"github.com/OWASP/Amass/amass/core"
+)
+
+const (
+	key = "my_api_key"
+	secret = "my_secret_key"
+)
+
+func TestTwitter(t *testing.T) {
+	if *networkTest == false {
+		return
+	}
+
+	config := setupConfig(domainTest)
+	
+	personalKey := &core.APIKey{
```

Comment:
I think there's a better way to handle API keys for unit tests. We should provide a configuration file flag to the tests and load the API keys from that.

Look into `sources/sources_test.go` for test flags and `LoadSettings()` in `core/config.go` for loading the configuration.

Reply:
I agree, I was thinking about doing something like that. 
I'm working on it, will update it soon.

Reply:
Should I store the test configuration file in the same folder (amass/sources/) ?

Reply:
The path to the config file will be provided by the user running the test via the command-line. Some of this will be implemented by others and require you to use placeholders for now.

Reply:
Updated. I'm now loading the same configuration file used for the the main program for the unit tests.